lib/ansible/modules/database/postgresql/postgresql_sequence.py

```diff
@@ -0,0 +1,745 @@
+    description:
+    - Path to a Unix domain socket for local connections.
+    type: str
+  ssl_mode:
+    description:
+    - Determines whether or with what priority a secure SSL TCP/IP connection
+      will be negotiated with the server.
+    - See U(https://www.postgresql.org/docs/current/static/libpq-ssl.html) for
+      more information on the modes.
+    - Default of C(prefer) matches libpq default.
+    default: prefer
+    choices: [ allow, disable, prefer, require, verify-ca, verify-full ]
+    type: str
+  ca_cert:
+    description:
+    - Specifies the name of a file containing SSL certificate authority (CA)
+      certificate(s).
+    - If the file exists, the server's certificate will be
+      verified to be signed by one of these authorities.
+    type: str
+    aliases: [ ssl_rootcert ]
+notes:
+- If you do not pass db parameter, sequence will be created in the database
+  named postgres.
+- The default authentication assumes that you are either logging in as or
+  sudo'ing to the postgres account on the host.
+- To avoid "Peer authentication failed for user postgres" error,
+  use postgres user as a I(become_user).
+- This module uses psycopg2, a Python PostgreSQL database adapter. You must
+  ensure that psycopg2 is installed on the host before using this module.
+- If the remote host is the PostgreSQL server (which is the default case), then
+  PostgreSQL must also be installed on the remote host. For Ubuntu-based
+  systems, install the postgresql, libpq-dev, and python-psycopg2 packages
+  on the remote host before using this module.
+requirements: [ psycopg2 ]
+author:
+- Tobias Birkefeld (@tcraxs)
+'''
+
+EXAMPLES = r'''
+- name: Create an ascending bigint sequence called foobar in the default
+        database
+  postgresql_sequence:
+    name: foobar
+
+- name: Create an ascending integer sequence called foobar, starting at 101
+  postgresql_sequence:
+    name: foobar
+    data_type: integer
+    start: 101
+
+- name: Create an descending sequence called foobar, starting at 101 and
+        preallocated 10 sequence numbers in cache
+  postgresql_sequence:
+    name: foobar
+    increment: -1
+    cache: 10
+    start: 101
+
+- name: Create an ascending sequence called foobar, which cycle between 1 to 10
+  postgresql_sequence:
+    name: foobar
+    cycle: yes
+    min: 1
+    max: 10
+
+- name: Create an ascending bigint sequence called foobar in the default
+        database with owner foobar
+  postgresql_sequence:
+    name: foobar
+    owner: foobar
+
+- name: Rename an existing sequence named foo to bar
+  postgresql_sequence:
+    name: foo
+    rename_to: bar
+
+- name: Change the schema of an existing sequence to foobar
+  postgresql_sequence:
+    name: foobar
+    newschema: foobar
+
+- name: Change the owner of an existing sequence to foobar
+  postgresql_sequence:
+    name: foobar
+    owner: foobar
+
+- name: Drop a sequence called foobar
+  postgresql_sequence:
+    name: foobar
+    state: absent
```

Comment:
If I try to drop non existent sequence it return
```
"msg": "Tries to drop nonexistent sequence 'foobar2'"
```
but should return changed=false
Please, fix it and add tests for it